src/index.html

```diff
@@ -2,11 +2,47 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
     <h1>Calendar</h1>
+    <div class="calendar calendar__day--sun">
```

Comment:
add some day count modifier to this classes and check is it working with all numbers